lib/routes/account.js

```diff
@@ -1363,6 +1363,25 @@ module.exports = function (
                 .catch(catchPushError)
             }
           })
+          .then(function () {
+            // Emit a metrics event for when a user sends tabs between devices.
+            // In the future we will aim to get this event directly from sync telemetry,
+            // but we're doing it here for now as a quick way to get metrics on the feature.
+            if (payload.command === 'sync:collection_changed') {
+              // Note that payload schema validation ensures that these properties exist.
+              if (payload.data.collections.length === 1 && payload.data.collections[0] === 'clients') {
```

Comment:
This looks correct for android. This is what it sends to devices/notify endpoint:
```
{
to: [...device_ids...],
TTL: 123,
payload: {
 version:1,
 command:"sync:collection_changed",
 data: {
  collections: ["clients"]
 }
}
}
```